lib/MHonArc/CharEnt/AppleArabic.pm

```diff
@@ -1,4 +1,7 @@
 package MHonArc::CharEnt::AppleArabic;
```

Comment:
Obviously, Github have some troubles with the characters in some files :shrug: 

If you checkout my branch and do `git diff a98d78e..e4b5f4a`, it should be displayed correctly (here, the first line is `package MHonArc::UTF8::AppleArabic;`), except for characters that you don't have the font to display.

Reply:
Anyways we may let homemade codecs retire: `Encode` module became standard way as of Perl 5.8. I'd like to work for this improvement in some day.

Reply:
@ikedas sounds good :+1: